hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -2544,8 +2571,12 @@ public void testBootstrapWithTableNotFound() throws Exception {
     // delete the metadata folder.
     fs.delete(metadataTablePath, true);
     assertFalse(fs.exists(metadataTablePath));
+    Set<String> removingPropKeys = removeMetadataPartitionsProps(metaClient.getTableConfig().getProps())
+        .getRight().keySet().stream().map(Object::toString).collect(Collectors.toSet());
+    Path metaPathDir = new Path(metaClient.getBasePathV2(), METAFOLDER_NAME);
+    HoodieTableConfig.delete(fs, metaPathDir, removingPropKeys);
```

Comment:
to be extracted

Reply:
this seems to be a new test case added from the RLI patch